lib/bootloader_spvm.pm

```diff
@@ -26,7 +26,7 @@ use registration 'registration_bootloader_params';
 use utils qw(get_netboot_mirror type_string_slow);
 
 our @EXPORT = qw(
-  boot_spvm
+  boot_pvm
```

Comment:
as you want to generalize this file then I suggest to also rename the complete file, e.g. `bootloader_pvm.pm`

Reply:
did so now with https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/9016/commits/8fa8318f2f78fbfd3665f2bee6295eafb7f3bd7e